### PR TITLE
Update stable/postgresql/README.md

### PR DESCRIPTION
**Pull request by AdamDang**
Delete dulicated word.
 Line 107: a Persistent Volume volume

### CODE REVIEW

**Comment by timstoop** [comment]
The reason that it's failing is that some of the requirements of the Chart.yaml have changed recently and this Chart is missing an appVersion key. You could add it and simply set it to the version of the image that's default. That would move this forward and fix it for future commits to this Chart as well. (Do I sound like a broken record yet? ;-) )

**Comment by AdamDang** [comment]
Thanks @timstoop I've add the appVersion key.

File: stable/postgresql/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name: postgresql
-version: 0.9.3
+version: 0.9.4
+appVersion: 0.9.4
```
Comment (sameersbn) [inline stable/postgresql/Chart.yaml]: @AdamDang the appVersion should reflect the version of the psql server installed by the chart which is `9.6.2`

**Review by sameersbn — CHANGES_REQUESTED** [review]

**Comment by AdamDang** [comment]
@sameersbn 
 I've modified the appVersion to 9.6.2. Thanks,

**Review by sameersbn — APPROVED** [review]

**Pull request merged**